core/src/components/item/item.scss

```diff
@@ -51,6 +51,15 @@
    * @prop --highlight-color-valid: The color of the highlight on the item when valid. Only applies to inputs and textareas using the legacy form syntax. DEPRECATED: Highlights can be styled on `ion-input` or `ion-textarea` when using the modern form syntax.
    * @prop --highlight-color-invalid: The color of the highlight on the item when invalid. Only applies to inputs and textareas using the legacy form syntax. DEPRECATED: Highlights can be styled on `ion-input` or `ion-textarea` when using the modern form syntax.
    */
+
+  /**
+   * We change the minimum width as the
+   * font size changes. Using a fixed minimum
+   * width means that fewer and fewer characters
+   * can be displayed in the same space as the
+   * text grows.
+   */
+  --inner-min-width: 4rem;
```

Comment:
Why do we need a CSS variable for this? Are we expecting users to update it?

Reply:
This is explained in the original post: 

> I made this a CSS variable but left it undocumented. If developers need a way of changing this min-width they can request it and we can easily expose this variable.

This was a way to minimize risk in case people are using narrow lists that are less than the minimum width. I'm fine removing it if you don't think it's needed.

Reply:
Oh oops I should've read that, sorry. 😆